ISSUE_TEMPLATE.md

```diff
@@ -10,6 +10,15 @@
 
 `python -c "import poliastro.testing; poliastro.testing.test()"`
 
+`pip freeze | grep astropy`
+
+`pip freeze | grep poliastro`
+
+<!--- If the "pip freeze" commands don't work, you likely dont have pip installed. -->
```

Comment:
Do you think there's a case where the user might not have `pip` installed?

Reply:
To be honest, most likely not.

Reply:
Then if you could please remove that part, I'll go ahead and merge this :)